src/Libraries/RevitNodes/Filter/OverrideGraphicSettings.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using Autodesk.Revit.DB;
+using DynamoServices;
+using Autodesk.DesignScript.Runtime;
+using Revit.GeometryConversion;
+using RevitServices.Persistence;
+using RevitServices.Transactions;
+using System.Collections.Generic;
+
+namespace Revit.Filter
+{
+    /// <summary>
+    /// Override Graphic Settings
+    /// </summary>
+    public class OverrideGraphicSettings
+    {
+
+        #region Internal Properties
+
+        /// <summary>
+        /// Internal reference to the Revit Element
+        /// </summary>
+        internal Autodesk.Revit.DB.OverrideGraphicSettings InternalOverrideGraphicSettings
+        { 
+            get; set;
+        }
+
+        /// <summary>
+        /// Reference to the Element
+        /// </summary>
+        internal OverrideGraphicSettings(Autodesk.Revit.DB.OverrideGraphicSettings internalOverrideGraphicSettings)
```

Comment:
this would be the same situation, Revit element wrapper like the parameter implementation.